src/ui/webeiddialog.hpp

```diff
@@ -26,6 +26,13 @@
 
 #include <QCloseEvent>
 
+#ifndef Q_DISABLE_COPY_MOVE
+#define Q_DISABLE_COPY_MOVE(Class) \
+    Q_DISABLE_COPY(Class) \
+    Class(Class &&) = delete; \
+    Class &operator=(Class &&) = delete;
+#endif
```

Comment:
Better to make this DRY as well, suggest moving this to `src/controller/utils.hpp`.

Reply:
This should be probably moved to libelectronic-id or even pcsc